tests/baselines/reference/unusedLocalsInMethod4.errors.txt

```diff
@@ -0,0 +1,191 @@
+unusedLocalsInMethod4.ts(15,21): error TS2454: Variable 'x1' is used before being assigned.
+unusedLocalsInMethod4.ts(16,21): error TS2454: Variable 'x2' is used before being assigned.
+unusedLocalsInMethod4.ts(22,21): error TS2454: Variable 'x8' is used before being assigned.
+unusedLocalsInMethod4.ts(23,21): error TS2454: Variable 'x9' is used before being assigned.
+unusedLocalsInMethod4.ts(44,17): error TS2454: Variable 'x1' is used before being assigned.
+unusedLocalsInMethod4.ts(45,17): error TS2454: Variable 'x2' is used before being assigned.
+unusedLocalsInMethod4.ts(51,17): error TS2454: Variable 'x8' is used before being assigned.
+unusedLocalsInMethod4.ts(52,17): error TS2454: Variable 'x9' is used before being assigned.
+unusedLocalsInMethod4.ts(58,9): error TS2454: Variable 'x' is used before being assigned.
+unusedLocalsInMethod4.ts(67,20): error TS2454: Variable 'x' is used before being assigned.
+unusedLocalsInMethod4.ts(103,11): error TS1155: 'const' declarations must be initialized.
+unusedLocalsInMethod4.ts(113,9): error TS2454: Variable 'i' is used before being assigned.
+unusedLocalsInMethod4.ts(114,21): error TS2454: Variable 'i' is used before being assigned.
+unusedLocalsInMethod4.ts(139,9): error TS2454: Variable 'enabledSubstitutions' is used before being assigned.
+unusedLocalsInMethod4.ts(141,9): error TS2454: Variable 'enabledSubstitutions' is used before being assigned.
+
+
+==== unusedLocalsInMethod4.ts (15 errors) ====
+    function f<T, NonNull extends {}>() {
+        let x1: number[]; // should error
+        let x2: number[] | null; // should error
+        let x3: number[] | undefined; // should not error
+        let x4: number[] | undefined | null; // should not error
+        let x5!: number[]; // should not error
+        let x6: any; // should not error
+        let x7: unknown; // should not error
+        let x8: T; // should error
+        let x9: NonNull; // should error
+        var x10: NonNull; // should error

```

Comment:
This `// should error` is out of date since we're not erroring on `var`s anymore.